2015/02/02 12:35:37 [error] 4219#0: *1 open() "/home/pierre/git/Django-ClasseInversee/ClasseInversee1/media/PICT0010.JPG" failed (13: Permission denied), client: 127.0.0.1, server: 127.0.0.1, request: "GET /media/PICT0010.JPG HTTP/1.1", host: "localhost:8000"
2015/02/02 12:38:07 [error] 4219#0: *2 open() "/home/pierre/git/Django-ClasseInversee/ClasseInversee1/static/Gambian_classroom.jpg" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static/Gambian_classroom.jpg HTTP/1.1", host: "localhost:8000"
2015/02/02 12:38:15 [error] 4219#0: *2 open() "/home/pierre/git/Django-ClasseInversee/ClasseInversee1/static_dep/Gambian_classroom.jpg" failed (2: No such file or directory), client: 127.0.0.1, server: 127.0.0.1, request: "GET /static_dep/Gambian_classroom.jpg HTTP/1.1", host: "localhost:8000"
2015/02/02 12:40:53 [error] 4219#0: *4 open() "/home/pierre/git/Django-ClasseInversee/ClasseInversee1/media/PICT0010.JPG" failed (13: Permission denied), client: 127.0.0.1, server: 127.0.0.1, request: "GET /media/PICT0010.JPG HTTP/1.1", host: "localhost:8000"
2015/02/02 12:52:21 [error] 4471#0: *1 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://127.0.0.1:8000", host: "localhost:8000"
2015/02/02 13:06:47 [error] 4471#0: *4 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://127.0.0.1:8000", host: "localhost:8000"
2015/02/02 13:06:51 [error] 4471#0: *4 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://127.0.0.1:8000", host: "localhost:8000"
2015/02/02 13:06:52 [error] 4471#0: *4 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://127.0.0.1:8000", host: "localhost:8000"
2015/02/02 13:07:06 [error] 4471#0: *11 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://127.0.0.1:8000", host: "127.0.0.1:8000"
2015/02/02 13:07:06 [error] 4471#0: *11 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://127.0.0.1:8000", host: "127.0.0.1:8000"
2015/02/02 13:09:40 [error] 4471#0: *16 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://127.0.0.1:8000", host: "127.0.0.1:8000"
2015/02/02 13:10:15 [error] 4471#0: *16 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://127.0.0.1:8000", host: "127.0.0.1:8000"
2015/02/02 13:10:32 [error] 4471#0: *16 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://127.0.0.1:8000", host: "127.0.0.1:8000"
2015/02/02 13:10:33 [error] 4471#0: *16 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://127.0.0.1:8000", host: "127.0.0.1:8000"
2015/02/02 13:12:33 [error] 4471#0: *25 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://127.0.0.1:8000", host: "127.0.0.1:8000"
2015/02/02 13:13:07 [error] 4471#0: *25 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://127.0.0.1:8000", host: "127.0.0.1:8000"
2015/02/02 16:30:30 [error] 5969#0: *7 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/02/02 16:33:38 [error] 5969#0: *9 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/02/02 16:33:39 [error] 5969#0: *9 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/02/02 16:33:44 [error] 5969#0: *9 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/02/02 16:34:31 [error] 5969#0: *9 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/02/02 16:34:31 [error] 5969#0: *9 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/02/05 11:10:08 [crit] 11198#0: *6 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /welcome HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000", referrer: "http://127.0.0.1:8000/"
2015/02/05 11:10:08 [crit] 11198#0: *6 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/02/05 11:10:10 [crit] 11198#0: *6 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /welcome HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000", referrer: "http://127.0.0.1:8000/"
2015/02/05 11:10:12 [crit] 11198#0: *10 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/06 10:02:30 [crit] 11198#0: *12 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/06 21:24:40 [crit] 11198#0: *14 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /register/ HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/02/06 21:28:44 [crit] 11198#0: *16 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 192.168.1.29, server: 127.0.0.1, request: "GET /welcome HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "192.168.1.89:8000"
2015/02/06 21:28:44 [crit] 11198#0: *16 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 192.168.1.29, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "192.168.1.89:8000"
2015/02/07 13:46:51 [crit] 11198#0: *19 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/07 13:46:51 [crit] 11198#0: *19 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/07 13:58:51 [crit] 2824#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/07 13:58:52 [crit] 2824#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/08 10:02:32 [crit] 2809#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/08 10:02:33 [crit] 2809#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/09 07:51:21 [crit] 2806#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/09 07:51:22 [crit] 2806#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/10 10:48:27 [crit] 2816#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/10 10:48:27 [crit] 2816#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/11 11:02:20 [crit] 2790#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/11 11:02:22 [crit] 2790#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/11 12:12:49 [crit] 2789#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/12 08:44:19 [crit] 2789#0: *3 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/12 08:44:20 [crit] 2789#0: *3 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/12 15:46:54 [crit] 2789#0: *6 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /admin/MonEtablissement/mesquestion/ HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/02/12 15:46:54 [crit] 2789#0: *6 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/02/12 15:46:54 [crit] 2789#0: *6 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/02/13 13:29:43 [crit] 2789#0: *10 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/13 13:29:44 [crit] 2789#0: *10 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/14 13:55:08 [crit] 2794#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/14 13:55:10 [crit] 2794#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/14 21:51:05 [crit] 2794#0: *4 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/14 21:51:06 [crit] 2794#0: *4 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/14 22:20:19 [crit] 2794#0: *7 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/14 22:20:20 [crit] 2794#0: *7 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/14 23:52:29 [crit] 2794#0: *10 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/14 23:52:31 [crit] 2794#0: *10 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/15 08:04:26 [crit] 2813#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/15 08:04:27 [crit] 2813#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/15 16:10:37 [crit] 2813#0: *4 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/15 16:10:38 [crit] 2813#0: *4 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/16 08:00:40 [crit] 2811#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/16 08:00:40 [crit] 2811#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/16 12:50:02 [crit] 2811#0: *4 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/16 12:50:02 [crit] 2811#0: *4 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/16 13:11:46 [crit] 2238#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/16 13:11:47 [crit] 2238#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/16 23:48:54 [crit] 2238#0: *4 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/16 23:48:55 [crit] 2238#0: *4 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/17 01:02:31 [crit] 2238#0: *7 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/17 01:02:31 [crit] 2238#0: *7 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/17 06:44:23 [crit] 2793#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/17 06:44:23 [crit] 2793#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/17 08:19:50 [crit] 2793#0: *4 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/17 08:19:50 [crit] 2793#0: *4 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/17 08:30:23 [crit] 2799#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/17 08:30:23 [crit] 2799#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/17 09:16:47 [crit] 2799#0: *5 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/17 09:16:47 [crit] 2799#0: *5 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "localhost:8000"
2015/02/17 09:18:47 [crit] 2799#0: *8 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /welcome HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000", referrer: "http://127.0.0.1:8000/"
2015/02/17 09:18:47 [crit] 2799#0: *8 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/02/17 09:18:47 [crit] 2799#0: *8 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/02/17 15:29:19 [crit] 2799#0: *13 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/02/17 15:29:20 [crit] 2799#0: *13 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/02/17 18:37:11 [crit] 2799#0: *16 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/02/17 18:37:12 [crit] 2799#0: *16 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/02/17 23:38:57 [crit] 2799#0: *20 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/02/17 23:38:57 [crit] 2799#0: *20 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/02/18 07:57:38 [crit] 2237#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/02/18 10:20:05 [crit] 2237#0: *6 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/02/18 10:20:06 [crit] 2237#0: *6 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/02/18 21:38:34 [crit] 2237#0: *9 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /welcome HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000", referrer: "http://127.0.0.1:8000/"
2015/02/26 09:34:52 [crit] 2234#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /welcome HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/02/26 09:34:52 [crit] 2234#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/02/26 09:49:03 [crit] 2234#0: *4 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /welcome HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/03/16 13:28:35 [crit] 1833#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /exampleform/ HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/03/16 13:28:35 [crit] 1833#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
2015/03/16 13:28:35 [crit] 1833#0: *1 connect() to unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET /favicon.ico HTTP/1.1", upstream: "uwsgi://unix:///home/pierre/git/Django-ClasseInversee/ClasseInversee1/mysite.sock:", host: "127.0.0.1:8000"
